### ISSUE TITLE
Unable to install python datatable on windows

### ISSUE THREAD

**Issue by n-srinidhi** [h2oai/datatable#3416]
Hi!I am trying to install python datatable on my windows machine using `pip install datatable`. The command throws the following error. I tried installing it within my virtual environment as well as without a virtual environment, but the issue persists. 
```
Python Version: 3.10.8
Windows Version: Windows 11
```

Error:
```
Traceback (most recent call last):
        File "C:\Users\srini\Desktop\Projects\Centra\centra\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 144, in prepare_metadata_for_build_wheel
          hook = backend.prepare_metadata_for_build_wheel
      AttributeError: module 'ext' has no attribute 'prepare_metadata_for_build_wheel'

      During handling of the above exception, another exception occurred:

      Traceback (most recent call last):
        File "C:\Users\srini\Desktop\Projects\Centra\centra\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 351, in <module>
          main()
        File "C:\Users\srini\Desktop\Projects\Centra\centra\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 333, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "C:\Users\srini\Desktop\Projects\Centra\centra\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 148, in prepare_metadata_for_build_wheel
          whl_basename = backend.build_wheel(metadata_directory, config_settings)
        File "C:\Users\srini\AppData\Local\Temp\pip-install-vvtt0lzm\datatable_5fe1a2a7477b4884b7bf17c72aec873d\ci\ext.py", line 587, in build_wheel
          so_file = build_extension(cmd=("debug" if debug_wheel else "build"),
        File "C:\Users\srini\AppData\Local\Temp\pip-install-vvtt0lzm\datatable_5fe1a2a7477b4884b7bf17c72aec873d\ci\ext.py", line 225, in build_extension
          ext.compiler.enable_colors()
        File "C:\Users\srini\AppData\Local\Temp\pip-install-vvtt0lzm\datatable_5fe1a2a7477b4884b7bf17c72aec873d\ci\xbuild\compiler.py", line 274, in enable_colors
          if not self.is_msvc():
        File "C:\Users\srini\AppData\Local\Temp\pip-install-vvtt0lzm\datatable_5fe1a2a7477b4884b7bf17c72aec873d\ci\xbuild\compiler.py", line 91, in is_msvc
          return self.flavor == "msvc"
        File "C:\Users\srini\AppData\Local\Temp\pip-install-vvtt0lzm\datatable_5fe1a2a7477b4884b7bf17c72aec873d\ci\xbuild\compiler.py", line 78, in flavor
          self._detect_compiler_executable()
        File "C:\Users\srini\AppData\Local\Temp\pip-install-vvtt0lzm\datatable_5fe1a2a7477b4884b7bf17c72aec873d\ci\xbuild\compiler.py", line 146, in _detect_compiler_executable
          self._detect_winsdk()
        File "C:\Users\srini\AppData\Local\Temp\pip-install-vvtt0lzm\datatable_5fe1a2a7477b4884b7bf17c72aec873d\ci\xbuild\compiler.py", line 207, in _detect_winsdk
          % msvc_path)
      NameError: name 'msvc_path' is not defined
      [end of output]
```
How can I resolve this?

**Comment by oleksiyskononenko** [comment]
For Python 3.10 there is no datatable release available on pypi, but you can still install the latest dev wheels from our AppVeyor artifacts: https://datatable.readthedocs.io/en/latest/start/install.html#install-latest-dev-version

There are wheels available for both Python 3.10 and 3.11.

**Comment by n-srinidhi** [comment]
This worked like a charm!Thanks a lot!
